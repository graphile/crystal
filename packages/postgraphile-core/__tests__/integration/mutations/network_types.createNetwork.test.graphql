#> schemas: ["network_types"]
#> options: { graphileBuildOptions: { pgUseCustomNetworkScalars: true } } 

mutation {
  createNetwork(input: {
    network: {
      inet: "10.0.0.0"
      cidr: "10.0.0.0/8"
      macaddr: "000000000000"
    }
  }) {
    network {
      nodeId
      id
      inet
      cidr
      macaddr
    }
  }
}
