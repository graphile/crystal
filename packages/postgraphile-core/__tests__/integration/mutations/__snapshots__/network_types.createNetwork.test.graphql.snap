// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "createNetwork": Object {
      "network": Object {
        "cidr": "10.0.0.0/8",
        "id": 7,
        "inet": "10.0.0.0",
        "macaddr": "00:00:00:00:00:00",
        "nodeId": "WyJuZXR3b3JrcyIsN10=",
      },
    },
  },
}
`;
